static/app/views/dashboards/widgets/common/widgetFrame.tsx

```diff
@@ -216,6 +238,23 @@ const RigidBadge = styled(Badge)`
   flex-shrink: 0;
 `;
 
+const WidgetTooltipTitle = styled('div')`
+  font-weight: bold;
+  font-size: ${p => p.theme.fontSizeMedium};
+  text-align: left;
+`;
+
+const WidgetTooltipDescription = styled('div')`
+  margin-top: ${space(0.5)};
+  font-size: ${p => p.theme.fontSizeSmall};
+  text-align: left;
+`;
+
+// We're using a button here to preserve tab accessibility
+const WidgetTooltipButton = styled(Button)`
```

Comment:
Can't we assign something like `tabindex` to be tab-able?

Reply:
Ah, yes, _but_ it's still most correct for the trigger to be a button, IMO. It would also be really good if _clicking_ this button permanently showed the tooltip until clicked again, TBH, but one thing at a time